mithril-aggregator/src/http_server/routes/aggregator_status.rs

```diff
@@ -0,0 +1,284 @@
+use warp::Filter;
+
+use mithril_common::{messages::AggregatorStatusMessage, StdResult};
+
+use crate::{
+    dependency_injection::EpochServiceWrapper,
+    http_server::routes::{middlewares, router::RouterState},
+};
+
+pub fn routes(
+    router_state: &RouterState,
+) -> impl Filter<Extract = (impl warp::Reply,), Error = warp::Rejection> + Clone {
+    status(router_state)
+}
+
+/// GET /status
+fn status(
+    router_state: &RouterState,
+) -> impl Filter<Extract = (impl warp::Reply,), Error = warp::Rejection> + Clone {
+    warp::path!("status")
+        .and(warp::get())
+        .and(middlewares::with_logger(router_state))
+        .and(middlewares::with_epoch_service(router_state))
+        .and(middlewares::extract_config(router_state, |config| {
+            config.cardano_node_version.clone()
+        }))
+        .and_then(handlers::status)
+}
+
+async fn get_aggregator_status_message(
+    epoch_service: EpochServiceWrapper,
+    cardano_node_version: String,
+) -> StdResult<AggregatorStatusMessage> {
+    let epoch_service = epoch_service.read().await;
+
+    let epoch = epoch_service.epoch_of_current_data()?;
+    let cardano_era = epoch_service.cardano_era()?;
+    let mithril_era = epoch_service.mithril_era()?;
+    let aggregator_node_version = env!("CARGO_PKG_VERSION").to_string();
```

Comment:
What happens if the environment variable is changed during execution? 
This must not happen it seems a bit dangerous to me.

Reply:
Using `env!("CARGO_PKG_VERSION")` is safe because it embeds the version at compile-time, it can't change during runtime.